grants_tagger/train.py

```diff
@@ -223,14 +224,44 @@ def train_and_evaluate(
             label_binarizer=label_binarizer,
             from_same_distribution=from_same_distribution
         )
-        model.fit(X_train, Y_train)
+        if Y_train.shape[1] > 512:
```

Comment:
this flag possibly needs a different name since neural nets do not need batching of Y. possibly y_batch_size.